support-frontend/app/services/stepfunctions/Client.scala

```diff
@@ -35,6 +36,10 @@ class Client(client: AWSStepFunctionsAsync, arn: StateMachineArn) {
 
   private def startExecution(arn: String, input: String)(implicit ec: ExecutionContext): Response[StartExecutionResult] = convertErrors {
     AwsAsync(client.startExecutionAsync, new StartExecutionRequest().withStateMachineArn(arn).withInput(input))
+      .transform { theTry =>
+        SafeLogger.info(s"state machine result: $theTry")
```

Comment:
the problem I had was the ARN was wrong in my local support-private config.  It didn't actually log that.  I don't think we need this logging, I think we should actually make `convertErrors` store the original error in the converted error so we can debug it properly.